pwa-devdocs/src/_includes/layout/pre-release-banner.html

```diff
@@ -1,4 +1,4 @@
 <div class="message-banner">
   The content on this site reflects the current progress of the Magento PWA project.
```

Comment:
I would remove this line to keep banner content to a minimum.
```suggestion
```

Reply:
Thanks for your rewording (and also for the fixing)! I also thought it's pretty long but couldn't come up with a better solution.

As for "The content on this site reflects...", as this text had been added before I didn't dare to touch it. I am not sure if this has to be kept to reflect that PWA studio is still a bit of WIP. @zetlen, please can you have a quick look here?

Reply:
Hey @mzeis, I'm catching up (wow, 28 day turnaround is a new record for me!) and I endorse @jcalcaben's recommendation here--especially since git blame says he wrote that in the first place.